python-lib/spacy_tokenizer.py

```diff
@@ -136,7 +162,22 @@ def _create_spacy_tokenizer(self, language: AnyStr) -> Language:
             if "#" in _prefixes:
                 _prefixes.remove("#")
                 nlp.tokenizer.prefix_search = spacy.util.compile_prefix_regex(_prefixes).search
-        logging.info("Loading tokenizer for language '{}': Done in {:.2f} seconds.".format(language, time() - start))
+        if self.stopwords_folder_path and language in SUPPORTED_LANGUAGES_SPACY:
+            try:
+                stopwords_file_path = os.path.join(self.stopwords_folder_path, f"{language}.txt")
+                with open(stopwords_file_path) as f:
+                    custom_stopwords = set(f.read().splitlines())
+                for word in custom_stopwords:
+                    nlp.vocab[word].is_stop = True
+                    nlp.vocab[word.capitalize()].is_stop = True
+                for word in nlp.Defaults.stop_words:
+                    if word not in custom_stopwords:
+                        nlp.vocab[word].is_stop = False
+                        nlp.vocab[word.capitalize()].is_stop = False
+                nlp.Defaults.stop_words = custom_stopwords
+            except OSError as e:
```

Comment:
There is something confusing here, if the user request the stopwords and there are no stopwords file, this should raise an Error otherwise you swallow it and the user is fooled

Reply:
Yeah you're right, it should fail

Reply:
Changed in e9b7f0ca8914a7ad8a58f22f4322d325da25f492